src/docs/development/tools/devtools/memory.md

```diff
@@ -149,86 +185,139 @@ in the legend.
 
 For more information, see [Dart VM internals][vm].
 
-## Event timeline
+## Memory Android Chart (ADB)
+
+DevTools, for an Android app, collects Android's ADB tool parsing the meminfo section of ADB's App Summary section.  This section is the most interesting from a high-level
+```
+> adb shell dumpsys meminfo io.flutter.demo.gallery -d
+
+ App Summary
+                       Pss(KB)
+                       -------
+           Java Heap:     5192
+         Native Heap:    11992
+                Code:     2132
+               Stack:       60
+            Graphics:    53700
+       Private Other:    42800
+              System:    84493
+ 
+               TOTAL:   200369       TOTAL SWAP PSS:    82168
+```
+
+This chart is another timeseries graph but of Android memory (polled ADB values).  to help visualize the state of the
```

Comment:
I think you can replace this whole sentence with "This chart is another timeseries graph."

the data the graph contains was already covered above

Reply:
Done.